include/deal.II/base/std_cxx11/type_traits.h

```diff
@@ -33,6 +33,8 @@ namespace std_cxx11
   using std::is_standard_layout;
   using std::is_trivial;
   using std::enable_if;
+  using std::true_type;
```

Comment:
Yeah, sorry about that. I didn't know that we have a script that did this automagically.